assets/js/data/validation/test/reducers.ts

```diff
@@ -0,0 +1,247 @@
+/**
+ * Internal dependencies
+ */
+import reducer, { FieldValidationStatus } from '../reducers';
```

Comment:
We have an import error: `Module '"../reducers"' has no exported member 'FieldValidationStatus'`. Let's import it from the types.ts file instead.